src/Castle.Services.Logging.NLogIntegration/Castle.Services.Logging.NLogIntegration-VS2017.csproj

```diff
@@ -22,7 +22,7 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="NLog" Version="4.4.5" />
+    <PackageReference Include="NLog" Version="4.4.0-beta13" />
```

Comment:
As discussed, let's keep NLog on the last version we shipped with `4.4.1`, and ignore .NET Core support because it doesn't really matter until NLog actually adds .NET Core support back in 5.x.